src/main/java/rx/schedulers/Schedulers.java

```diff
@@ -189,4 +189,17 @@ public static void shutdown() {
             RxRingBuffer.SPMC_POOL.shutdown();
         }
     }
+    
+    /**
+     * Enable or disable worker tracking; if a scheduled task crashes,
+     * the error reported to the RxJavaPlugins will contain an IllegalStateException
+     * with a CompositeException cause of the original crash exception 
+     * along with a RuntimeException holding the
+     * stacktrace where the parent worker has been instantiated.
+     * @param value the state
+     */
+    @Experimental
+    public static void setWorkerTracking(boolean value) {
```

Comment:
I find this API name non-obvious, asked 4 team members "what such method could do in their opinion" and only one had assumption that it may hold stacktrace, others assumed that it's some instrumentation api for RxJava scheduler workers.

My options at the moment are:

* `Schedulers.collectStacktraces()`
* `Schedulers.trackStacktraces()`
* `Schedulers.keepStacktraces()`